tools/theme/colors.ts

```diff
@@ -9,6 +9,7 @@ const colors = {
   secondary: {
     critical: '#DF1414',
     criticalHover: '#B21010',
+    successHover: '#1e7e34',
```

Comment:
Chose a random green color for the success hover. Will verify with designers what color they want